mgmt/rest/log_handler.go

```diff
@@ -50,6 +52,17 @@ func (l *Logger) ServeHTTP(rw http.ResponseWriter, r *http.Request, next http.Ha
 		"method":      r.Method,
 		"url":         r.URL.Path,
 		"status-code": res.Status(),
-		"status":      http.StatusText(res.Status()),
+		"status":      deprecated(http.StatusText(res.Status()), rw.Header().Get("Deprecated"), rw.Header().Get("DeprWhat"), rw.Header().Get("DeprInfo")),
```

Comment:
@aarontay,  what do you think about:
 - logging about apiV1 depreciation as a warning, not debug
- do it a separate log  (do not combine api response + deprecation info in one line)

**Why**: All of this to increase readability + keep the same pattern which has been used in RPC deprecation.

#### How it looks currently:
   For example: `$snaptel plugin load <plugin_path>`
    
   ![image](https://cloud.githubusercontent.com/assets/11335874/26327867/2eb08348-3f41-11e7-8a9d-9c1bc7f03a3a.png)

#### Suggestion:
   Separate log for deprecation info on warning level - done in a similar way as for deprecated RPC protocol, see below:
![image](https://cloud.githubusercontent.com/assets/11335874/26328003/a89c4a5c-3f41-11e7-979f-a4676a385a09.png)